### ISSUE TITLE
Is the fine-tuned I3D model on UCF-101 available ?

### ISSUE THREAD

**Issue by pinkfloyd06** [tomrunia/PyTorchConv3D#2]
Hello @tomrunia,

Le me first thank you for you work.

l'm looking for the fine-tuned I3D kinetics-400 model on UCF-101. Is it available ?

Thank you

**Comment by tomrunia** [comment]
I don't think so. Only for Charades and ImageNet: https://github.com/piergiaj/pytorch-i3d/tree/master/models
